themes/shared/single-post.scss

```diff
@@ -7,10 +7,12 @@
   .avatar {
     float: left;
     line-height: 0;
+    @media (max-width: 767px) { margin: 0 10px 10px 0; }
```

Comment:
I believe `0 10px 5px 0` looks better (along with the change in `.title`, see below). Could you please check it?